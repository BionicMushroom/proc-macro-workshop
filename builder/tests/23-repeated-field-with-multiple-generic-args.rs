use derive_builder::Builder;

struct Vec<T, U> {
    field0: T,
    field1: U,
}

#[derive(Builder)]
struct Test {
    #[builder(each = "whatever")]
    field: Vec<u32, u32>,
}

fn main() {}
