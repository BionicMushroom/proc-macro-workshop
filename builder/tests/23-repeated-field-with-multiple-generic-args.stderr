error: expected standard Vec
  --> tests/23-repeated-field-with-multiple-generic-args.rs:11:12
   |
11 |     field: Vec<u32, u32>,
   |            ^^^^^^^^^^^^^
